client/common/directives/ModelFieldPointsOfInterest/ModelFieldPointsOfInterest.js

```diff
@@ -65,22 +65,20 @@ angular.module('dashboard.directives.ModelFieldPointsOfInterest', [
            <option value="5" label="5 Miles">5 Miles</option> \
            <option value="10" label="10 Miles">10 Miles</option> \
            <option value="20" label="20 Miles">20 Miles</option> \
-        	 <option value="30" label="30 Miles">30 Miles</option> \
-         </select> \
-        <button class="btn btn-default" ng-click="doSearch()" ng-model="request.query" ng-disabled="disabled">Search</button><span class="search-error" ng-if="searchError">{{searchError}}</span>\
-        </accordion-group>\
-        </accordion> \
-        <div class="map-canvas"id="map_canvas"></div> \
-        <ul class="selected-location" ng-model="displayedSearchResults" > \
```

Comment:
I think we can probably get rid of/refactor some of the code related to this list.
- displayedSearchResults
- repeatExpression
- updateSelection

Reply:
repeatExpression, updateSelection was removed.
"displayedSearchResults" is still being used.

Reply:
I think displayedSearchResults is to support the checkboxes that were removed right? Along with the click handler and listSearchResults().

Reply:
Yeah you're right! Already removed those code blocks on the recent update. Thanks!